src/main/java/uk/gov/digital/ho/hocs/templates/TemplateService.java

```diff
@@ -99,9 +99,12 @@ private TeamDto getPrivateOfficeTeamDetails(UUID caseUUID, CaseDataDto caseDetai
     }
 
     private String formatDate(String unformattedDate) {
-        DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("dd MMMM yyyy");
-        LocalDate date = LocalDate.parse(unformattedDate);
-        return date.format(dateTimeFormatter);
+        if(unformattedDate != null){
+            DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("dd MMMM yyyy");
```

Comment:
The `DateTimeFormatter` should be a constant. Is this pattern used elsewhere and if so is there a good place to put the constant instead of inside this class?